CIXClient/src/TableBase.m

```diff
@@ -389,6 +389,7 @@ -(NSString *)description
     return text;
 }
 
+#if RETURN_CHAR_STAR
```

Comment:
What is this definition for?

Reply:
It's just my way of disabling some code.  RETURN_CHAR_STAR is not defined anywhere, so the     code between #if and #else can be deleted.  Sorry about that.